visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.ts

```diff
@@ -96,8 +86,33 @@ export class ArtificialIntelligenceController
 		}
 	}
 
-	applyCustomConfig(configId: string) {
-		CustomConfigHelper.applyCustomConfig(configId, this.storeService, this.threeCameraService, this.threeOrbitControlsService)
+	applySuspiciousMetric(metric: MetricSuggestionParameters, isOutlier: boolean) {
```

Comment:
I would rather remove the second parameter of `applySuspiciousMetric()`. We can get this information from the `MetricSuggestionParameters`.

Reply:
The problem is that we need to pass an additional parameter into the method, because the user may want to look at a suspicious metric or the very high risk view (only colors are changed). A suspicious metric may have `true` in `isOutlier`, but the user only wants to see the normal suspicious view. Do you maybe have an idea for this problem?

Reply:
Ah, I just noticed that very high risk configs do not exist asindependent configs. In Line 228 the `isOutlier` flag of an existing config is set.